pwiz_tools/Skyline/Model/AuditLog/AuditLogStrings.Designer.cs

```diff
@@ -1,1643 +1,1667 @@
-        }
-        
-        /// <summary>
-        ///   Looks up a localized string similar to Start of audit log for already existing document.
-        /// </summary>
-        public static string start_log_existing_doc {
-            get {
-                return ResourceManager.GetString("start_log_existing_doc", resourceCulture);
-            }
-        }
-        
-        /// <summary>
-        ///   Looks up a localized string similar to  -- .
-        /// </summary>
-        public static string TabSeparator {
-            get {
-                return ResourceManager.GetString("TabSeparator", resourceCulture);
-            }
-        }
-        
-        /// <summary>
-        ///   Looks up a localized string similar to {0}.
-        /// </summary>
-        public static string test_only {
-            get {
-                return ResourceManager.GetString("test_only", resourceCulture);
-            }
-        }
-        
-        /// <summary>
-        ///   Looks up a localized string similar to True.
-        /// </summary>
-        public static string True {
-            get {
-                return ResourceManager.GetString("True", resourceCulture);
-            }
-        }
-        
-        /// <summary>
-        ///   Looks up a localized string similar to The document was modified outside of Skyline.
-        /// </summary>
-        public static string undocumented_change {
-            get {
-                return ResourceManager.GetString("undocumented_change", resourceCulture);
-            }
-        }
-        
-        /// <summary>
-        ///   Looks up a localized string similar to Unrecognized charge states.
-        /// </summary>
-        public static string UnrecognizedChargeStates {
-            get {
-                return ResourceManager.GetString("UnrecognizedChargeStates", resourceCulture);
-            }
-        }
-        
-        /// <summary>
-        ///   Looks up a localized string similar to Unrecognized files.
-        /// </summary>
-        public static string UnrecognizedFiles {
-            get {
-                return ResourceManager.GetString("UnrecognizedFiles", resourceCulture);
-            }
-        }
-        
-        /// <summary>
-        ///   Looks up a localized string similar to Unrecognized peptides.
-        /// </summary>
-        public static string UnrecognizedPeptides {
-            get {
-                return ResourceManager.GetString("UnrecognizedPeptides", resourceCulture);
-            }
-        }
-        
-        /// <summary>
-        ///   Looks up a localized string similar to Upgraded background proteome &quot;{0}&quot;.
-        /// </summary>
-        public static string upgraded_background_proteome {
-            get {
-                return ResourceManager.GetString("upgraded_background_proteome", resourceCulture);
-            }
-        }
-    }
-}
+﻿//------------------------------------------------------------------------------

```

Comment:
Why is this a complete diff of the entire file?

Reply:
Fixed it.  I am still unsure as to how the line endings changed in the first place